usermembership/tool/src/webapp/usermembership/userlist.jsp

```diff
@@ -30,48 +31,40 @@
-			<h:panelGrid styleClass="sectionContainerNav" columns="2">
+		<t:div styleClass="page-header">
+			<h1><h:outputText value="#{msgs.title_userlist}"/></h1>
+		</t:div>
+		<sakai:instruction_message value="#{msgs.instructions_userlist}" />
+
+		<t:div styleClass="form-group row">
+			<t:div styleClass="col-sm-4">
 				<h:outputLabel for="selectType" value="#{msgs.combo_user_type}"/>
-				<t:selectOneMenu 
-	    	            id="selectType"
-			           	immediate="true"
-			           	value="#{UserListBean.selectedUserType}"
-			           	title="#{msgs.combo_user_type_title}"
-			       		style="width: 160px;">
-	    	        <f:selectItems value="#{UserListBean.userTypes}"/> 
-		        </t:selectOneMenu> 	
-	            <h:outputLabel for="selectAuthority" value="#{msgs.combo_user_ext}"/>
-	            <t:selectOneMenu 
-	    	           	id="selectAuthority"
-			       		immediate="true"
-			       		value="#{UserListBean.selectedAuthority}"
-			       		title="#{msgs.combo_user_ext_title}"
-			       		style="width: 160px;">
-	    	        <f:selectItems value="#{UserListBean.userAuthorities}"/> 
-		        </t:selectOneMenu>
-			</h:panelGrid>
-			
-			<t:div styleClass="act" style="padding: 0em;">
+				<t:selectOneMenu styleClass="form-control" id="selectType" immediate="true" value="#{UserListBean.selectedUserType}" title="#{msgs.combo_user_type_title}">
+					<f:selectItems value="#{UserListBean.userTypes}"/> 
+				</t:selectOneMenu>
+				<h:outputLabel for="selectAuthority" value="#{msgs.combo_user_ext}"/>
+				<t:selectOneMenu styleClass="form-control" id="selectAuthority" immediate="true" value="#{UserListBean.selectedAuthority}" title="#{msgs.combo_user_ext_title}">
+					<f:selectItems value="#{UserListBean.userAuthorities}"/> 
+				</t:selectOneMenu>
+			</t:div>
+			<t:div styleClass="col-sm-8">
+				<h:outputLabel for="inputSearchBox" value="#{msgs.bar_input_search_title}"/>
 				<h:inputText id="inputSearchBox" value="#{UserListBean.searchKeyword}"
 					valueChangeListener="#{UserListBean.processActionSearchChangeListener}"
-					size="20" title="#{msgs.bar_input_search_title}"
-					onfocus="if(this.value == '#{msgs.bar_input_search_inst}') this.value = '';"/>
-				<h:commandButton id="searchButton" action="#{UserListBean.processActionSearch}"
-					onkeypress="document.forms[0].submit;" value="#{msgs.bar_search}"
-					styleClass="active" style="margin-left: 3px !important;"
-					onclick="SPNR.disableControlsAndSpin( this, null );" />
-				<h:commandButton id="clearSearchButton" action="#{UserListBean.processActionClearSearch}"
-					onkeypress="document.forms[0].submit;" value="#{msgs.bar_clear_search}"
-					rendered="#{UserListBean.renderClearSearch}" style="margin-left: 3px !important;"
-					onclick="SPNR.disableControlsAndSpin( this, null );" />					
-	        </t:div>
-	 	</h:panelGrid>
-		
```

Comment:
Although updating the UI is a bit out of the scope of this issue, if you are fixing the layout of the table search and filters, please use the existing sakai-table-toolBar classes that take care of the layout and responsiveness for you. Worksite Setup's main page is a good example of this in use, which was added in #5865 based on the work in #5744. 

I can submit a separate Jira issue for this if you deem it to be out-of-scope for this PR.

Reply:
I agree it's super out of the scope of the issue but I think that if I have to work on something anyways and I have the chance to upgrade an old UI for something better it's worth the effort, I have to work in the tool anyway to fix the issue and we don't see the same UI again and again, Sakai is super big so working on improving small chuncks everytime we visit them it's just fine.

I'm fine reverting all the UX stuff and just fixing the bug, I've splitted the PR in three separate commits indicating bugfix, JS improvement about the search input and the UX improvements.

I tried to use the work of #5865 and I think looks worse but maybe I messed up with something:
![image](https://user-images.githubusercontent.com/8653754/131793193-00119f60-a3ec-4aa3-ab86-b07538f3e9e4.png)

vs
![image](https://user-images.githubusercontent.com/8653754/131792888-8566c66f-93b4-41af-852d-ba4939cf801f.png)

https://github.com/mpellicer/sakai-apereo/commit/a901f026c0bfc11ef81b5aa92f559b57ccefc053

Reply:
Okay. Stick with your original design and we can update it later. Thanks.

Reply:
@fostersdesign I'm fine investing more time on this! do you see anything wrong in the separate branch applying the stules?

Reply:
Sorry, where do I find that branch?

Reply:
https://github.com/mpellicer/sakai-apereo/commit/a901f026c0bfc11ef81b5aa92f559b57ccefc053

Reply:
Thanks. I'll take a look and let you know.

Reply:
You're really close. You just need to:
- remove the buttonContainer div (not needed if you don't have any buttons to put in it)
- move sakai-table-pagerContainer outside of sakai-table-filterContainer

Looks like we could add a whitespace: nowrap to the .sakai-table-toolBar .sakai-table-filterContainer label rule too:

`.sakai-table-toolBar .sakai-table-filterContainer label {
  white-space: nowrap;
}`

in _tables.scss (/library/src/morpheus-master/sass/modules/tool/_table.scss line 76)

Reply:
Thank you Shawn! Looks way better:

![image](https://user-images.githubusercontent.com/8653754/132012757-4a2296b0-1d16-458a-9744-7c5a37452cb9.png)

Reply:
Awwww deleted the branch by mistake, new PR here: https://github.com/sakaiproject/sakai/pull/9754